src/components/drawer/drawer.style.js

```diff
@@ -59,6 +65,12 @@ const StyledDrawerSidebar = styled(Box)`
   &::-webkit-scrollbar {
     width: 12px;
   }
+
+  color: var(--colorsUtilityYin090);
```

Comment:
Saying as this is duplicated from above, you could create a constant like 

```js
const drawerFont = `
  color: var(--colorsUtilityYin090);
  font-family: var(--fontFamiliesDefault);
  font-weight: var(--fontWeights400);
  font-size: var(--fontSizes100);
  line-height: var(--lineHeights500);
`;
```

and just put `${drawerFont}` here and above in the styled-components

Reply:
Nice simple idea. Good shout 😄